doc/api/url.md

```diff
@@ -250,7 +250,444 @@ properties of URL objects:
+[`url.parse()`][] and [`url.format()`][] methods would produce.
+
+#### url.toString()
+
+The `toString()` method on the `URL` object returns the serialized URL. The
+value returned is equivalent to that of `url.href`.
+
+### Class: URLSearchParams
+
+The `URLSearchParams` object provides read and write access to the querystring
+of a `URL`.
+
+```js
+const myURL = new URL('https://example.org/?abc=123');
+console.log(myURL.searchParams.get('abc'));
+  // Prints 123
+
+myURL.searchParams.append('abc', 'xyz');
+console.log(myURL.href);
+  // Prints https://example.org/?abc=123&abc=xyz
+
+myURL.searchParams.delete('abc');
+myURL.searchParams.set('a', 'b');
+console.log(myURL.href);
+  // Prints https://example.org/?a=b
+```
+
+#### Constructor: new URLSearchParams([init])
+
+* `init` {String} The URL querystring
+
+#### urlSearchParams.append(name, value)
+
+* `name` {String}
+* `value` {String}
+
+Append a new name-value pair to the query string.
+
+#### urlSearchParams.delete(name)
+
+* `name` {String}
+
+Remove all name-value pairs whose name is `name`.
+
+#### urlSearchParams.entries()
+
+* Returns: {Iterator}
+
+Returns an ES6 Iterator over each of the name-value pairs in the query string.
+Each item of the iterator is a JavaScript Array. The first item of the Array
+is the `name`, the second item of the Array is the `value`.
+
+Alias for `urlSearchParams\[\@\@iterator\]()`.
+
+#### urlSearchParams.get(name)
+
+* `name` {String}
+* Returns: {String} or `null` if there is no name-value pair with the given
+  `name`.
+
+Returns the value of the first name-value pair whose name is `name`.
+
+#### urlSearchParams.getAll(name)
+
+* `name` {String}
+* Returns: {Array}
+
+Returns the values of all name-value pairs whose name is `name`.
+
+#### urlSearchParams.has(name)
+
+* `name` {String}
+* Returns: {Boolean}
+
+Returns `true` if there is at least one name-value pair whose name is `name`.
+
+#### urlSearchParams.keys()
+
+* Returns: {Iterator}
+
+Returns an ES6 Iterator over the names of each name-value pair.
+
+#### urlSearchParams.set(name, value)
+
+* `name` {String}
+* `value` {String}
+
+Remove any existing name-value pairs whose name is `name` and append a new
+name-value pair.
+
+#### urlSearchParams.values()
+
+* Returns: {Iterator}
+
+Returns an ES6 Iterator over the values of each name-value pair.
+
+#### urlSearchParams\[\@\@iterator\]()
+
+* Returns: {Iterator}
+
+Returns an ES6 Iterator over each of the name-value pairs in the query string.
+Each item of the iterator is a JavaScript Array. The first item of the Array
+is the `name`, the second item of the Array is the `value`.
+
+Alias for `urlSearchParams.entries()`.
+
+### require('url').domainToAscii(domain)
+
+* `domain` {String}
+* Returns: {String}
+
+Returns the [Punycode][] ASCII serialization of the `domain`.
+
+*Note*: The `require('url').domainToAscii()` method is introduced as part of
+the new `URL` implementation but is not part of the WHATWG URL specification.
```

Comment:
The doc for `domainToUnicode` function uses the word "standard," which is the wording the URL Standard uses as well.